peers/connect_peer.js

```diff
@@ -58,7 +79,7 @@ module.exports = ({id, lnd}, cbk) => {
           return cbk();
         }
 
-        return asyncDetectSeries(getSockets.sockets, ({socket}, cbk) => {
+        return asyncDetectSeries(getSockets.sockets, (socket, cbk) => {
```

Comment:
isn't there a mismatch here, like the result of getNode sockets is sockets->socket but the argument passed directly is an array of socket